CHANGELOG.md

```diff
@@ -7,6 +7,29 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [template_v2.0.0] - unreleased
```

Comment:
Just a reminder to update `unreleased` to the actual date when this was released.